src/vrapp/VRVolumeApp.cpp

```diff
@@ -0,0 +1,1027 @@
+#include "vrapp/VRVolumeApp.h"
+#include "render/VolumeSliceRenderer.h"
+#include "render/VolumeRaycastRenderer.h"
+#include "render/Volume.h"
+#include "render/VolumeRenderer.h"
+#include "loader/LoadDescriptionAction.h"
+#include "loader/LoadNrrdAction.h"
+#include "loader/LoadDataAction.h"
+#include "UI/UIView.h"
+#include "interaction/HelperFunctions.h"
+#include "render/FontHandler.h"
+#include "interaction/CreateMovieAction.h"
+
+#include "GL/glew.h"
+#include <filesystem>
+#include <fstream> 
+#include <sstream> 
+#include "GLMLoader.h"
+
+#ifdef _MSC_VER
+#define _CRT_SECURE_NO_WARNINGS
+#include <windows.h>
+#define OS_SLASH_LOCAL "\\"
+#include "../../external/msvc/dirent.h"
+#else
+#define OS_SLASH_LOCAL "//"
+#include <dirent.h>
+#endif
+
+#include <glm/gtx/euler_angles.hpp>
+
+
+VRVolumeApp::VRVolumeApp():m_mesh_model(nullptr), m_clip_max{ 1.0f }, m_clip_min{ 0.0f }, m_clip_ypr{ 0.0f }, m_clip_pos{ 0.0 }, m_wasd_pressed(0),
```

Comment:
the formatting on this makes it a bit hard to read - can it be prettified?

Reply:
I formatted the Whole Project. 
pre...tti..fi..ed ? what's that?